cli/src/tasks/run.ts

```diff
@@ -19,16 +21,33 @@ import { getPlatformTargets } from '../util/native-run';
 import { sync } from './sync';
 
 export interface RunCommandOptions {
+  scheme?: string;
+  flavor?: string;
   list?: boolean;
   target?: string;
   sync?: boolean;
 }
 
 export async function runCommand(
-  config: Config,
+  cfg: Config,
   selectedPlatformName: string,
   options: RunCommandOptions,
 ): Promise<void> {
+  const toMerge: {
+    android?: {
+      flavor: string;
+    };
+    ios?: {
+      scheme: string;
+    };
+  } = {};
+  if (options.flavor) {
+    toMerge.android = { flavor: options.flavor };
+  }
+  if (options.scheme) {
+    toMerge.ios = { scheme: options.scheme };
+  }
+  const config: Config = merge(cloneDeep(cfg), toMerge);
```

Comment:
Seems like a little overkill to introduce lodash to change the config. Im cool with it but it does affect a users node_modules folder everytime a dependency is added

Reply:
Config is read-only so this lets us take a deep clone of the object then merge deep safely without changing the properties of Config type, this also only includes the bare minimum as not all of Lodash is installed

Reply:
You don't really need to clone and merge the config, what you can do is on the ios/android run.ts files read the scheme/flavor from the options and set a new scheme/flavor variable, something like
```
{ target: selectedTarget, flavor: selectedFlavor }: RunCommandOptions,
...
const runFlavor = selectedFlavor || config.android?.flavor || '';
...
 const arg = `assemble${runFlavor}Debug`;
```